osu.Game/Skinning/Editor/SkinBlueprint.cs

```diff
@@ -118,4 +127,58 @@ protected override void Update()
 
         public override Quad SelectionQuad => drawable.ScreenSpaceDrawQuad;
     }
+
+    internal class AnchorOriginVisualiser : CompositeDrawable
+    {
+        private readonly Drawable drawable;
+
+        private readonly Box originBox;
+
+        private readonly Box anchorBox;
+        private readonly Box anchorLine;
+
+        public AnchorOriginVisualiser(Drawable drawable)
+        {
+            this.drawable = drawable;
+
+            InternalChildren = new Drawable[]
+            {
+                anchorLine = new Box
+                {
+                    Colour = Color4.Yellow,
+                    Height = 2,
+                },
```

Comment:
Not 100% on why this is not a path (would save on the atan math) but I guess it does seem to work for now.